File: backend/app/models/review.py
from datetime import datetime


def new_review(
    author_name: str,
    email: str,
    rating: int,
    body: str,
    project_id: str | None = None,
) -> dict:
    return {
        "author_name": author_name,
        "email": email,
        "rating": rating,
        "body": body,
        "is_approved": False,
        "project_id": project_id,
        "created_at": datetime.utcnow(),
    }

File: backend/src/models/review.ts
export function newReview(
  authorName: string,
  email: string,
  rating: number,
  body: string,
  projectId: string | null = null,
) {
  return {
    author_name: authorName,
    email,
    rating,
    body,
    is_approved: false,
    project_id: projectId,
    created_at: new Date(),
  };
}
